### DISCUSSION TITLE
Can't select additional compatible game versions using terminal interface

### DISCUSSION THREAD

**Discussion by internetboyfriend666** [KSP-CKAN/CKAN#3614] [Q&A] [answered]
I'm using macOS 12.5 (Monterey) so I'm forced to use CKAN in terminal. I want to install the RSS/RO/RP1 suite of mods for KSP version 1.12.3, but in order to do so, the instructions say in CKAN I need to mark 1.11 as an additional compatible game version. I don't see any way I can do that in terminal. Hitting F10 does not give me anything that appears to be a way to do this. I would appreciate if anyone could help me out with this.

**Comment by HebaruSan** [comment, marked as answer]
Luckily I have it pinned to [the #macos channel in Discord](https://discord.gg/Mb4nXQD) for easy access:

> F10 → Select game instance... → Pick your install → Press 'e' to edit → Tab a few times to get to the compatible version list 

Though you can skip the second through fourth steps with this new option:

![image](https://user-images.githubusercontent.com/1559108/181683214-ce8dda10-0f12-4dc6-9aa5-bd8b16492f47.png)

There's also a section on the wiki user guide about it:

https://github.com/KSP-CKAN/CKAN/wiki/User-guide#choosing-compatible-game-versions-1

**Reply by internetboyfriend666** [reply]
Thank you! I didn't even know that channel existed
